tests/10-migration-backend-tests/verification/verify_migrations_test.go

```diff
@@ -15,6 +13,9 @@ import (
 	"path/filepath"
 	"strings"
 	"testing"
+
```

Comment:
Something interesting going on with import formatting I noticed in a couple of places.  Not too big of deal, but it'll appear in diffs as they go back and forth between developers.

Reply:
Hmmm... I'll look at this and can back this out. I generally don't like adding blank lines, so I'll see if something about go did that.

Reply:
I figured out what was happening here. I recently started to use SpaceVim which has a plugin for go. It tends to change these lines.  I've been using GoLand as well and will do that from now on since it doesn't try to change these lines. I'll revert this particular change above.

Reply:
Ok, yeah I don't mean for anyone to switch editors because of this.  I would imagine it is configurable behavior.  It looks like SpaceVim is grouping the imports, separating the Go standard imports from third-party or vendored imports.  I think Goland just alphabetizes them.